pyproject.toml

```diff
@@ -70,6 +70,7 @@ litellm = 'litellm:run_server'
 flake8 = "^6.1.0"
 black = "^23.12.0"
 pytest = "^7.4.3"
+pytest-mock = "^3.12.0"
```

Comment:
@vilmar-hillow do we need to add this as a required dependency for the sdk?

Reply:
> @vilmar-hillow do we need to add this as a required dependency for the sdk?

It's under `dev` dependency group, so that's where I'd usually expect it to be added, and your README references installing test dependencies via poetry, which also makes it a valid choice. However, your testing setup is not straightforward, so if there's a better place to add this dependency, let me know

Reply:
oh i misread the location. this is fine, thanks!